### PR TITLE
[LOOP-4975] Update Open Loop Freshness Logic and Labeling

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4975

### CODE REVIEW

File: Loop/Managers/ExtensionDataManager.swift [hunk 1]
```diff
@@ -118,6 +118,8 @@ final class ExtensionDataManager {
         #endif
 
         context.lastLoopCompleted = loopDataManager.lastLoopCompleted
+        context.lastCGMComm = loopDataManager.mostRecentGlucoseDataDate
```
Comment (ps2) [inline Loop/Managers/ExtensionDataManager.swift]: nit: the name indicates the last time of *comms*, but the value tracked is actually the date of the most recent data. May cause a future dev to make an incorrect assumption that the value is really the last comms date.

**Review by ps2 — APPROVED** [review]
LGTM, just a naming nit.

File: Loop/Managers/ExtensionDataManager.swift (hunk 1 above)
Comment (Camji55) [reply Loop/Managers/ExtensionDataManager.swift]: Gotcha, so it's possible that there's a comm from the pump that would not be "good" data? Is `mostRecentXDataDate` a good indicator of having no device issues?

Comment (Camji55) [reply Loop/Managers/ExtensionDataManager.swift]: Okay, naming updated.

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**